lib/generators/apple_auth/apple_sign_in_controller/templates/apple_sign_in_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class <%= @scope_prefix %>AppleSignInController < DeviseTokenAuth::SessionsController
+  protect_from_forgery with: :null_session
+  skip_before_action :verify_authenticity_token
+  before_action :skip_session_storage
+  before_action :check_json_request
+
+  def apple_sign_in
```

Comment:
maybe this method should be named `create`, I don't know what do the others think